Cargo.toml

```diff
@@ -547,5 +547,9 @@ name = "wasm"
 harness = false
 required-features = ["transforms-wasm", "transforms-lua"]
 
+[[bench]]
```

Comment:
Thanks for the heads-up.

I made the necessary changes. I did move `remap` back into its own benchmark target though, as it made testing/running benchmarks much faster for me, as I didn't need to enable unneeded features, and could just run `cargo bench --bench remap`.

We can change this back in a few weeks, if you like (I'm assuming this was done to reduce compile times on the CI), once we've added more benchmarks to Remap, and we can rely more on the CI to run them, instead of running them locally.